operator/cmd/seldon/cli/model_unload.go

```diff
@@ -49,14 +48,25 @@ func createModelUnload() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			modelName := args[0]
+			filename, err := flags.GetString(flagFile)
+			if err != nil {
+				return err
+			}
+			var fileBytes []byte
+			if filename != "" {
+				fileBytes = loadFile(filename)
+			}
+			modelName := ""
+			if len(args) > 0 {
+				modelName = args[0]
+			}
```

Comment:
these blocks of code look that they can be extracted as a helper for the 3 use cases (pipeline / model / experiment).